shared/desktop/app/index.js

```diff
@@ -117,6 +117,15 @@ function start() {
     }
   })
 
+  ipcMain.on('getMimeTypeFromURL', (event, url) => {
```

Comment:
Is this really necessary. We can't fetch from the main thread ? Or just use remote ?

Reply:
https://electronjs.org/docs/api/remote

Reply:
![mind-blowing](https://user-images.githubusercontent.com/255797/40193077-0cbb25fa-59bb-11e8-9c68-e819ec969679.gif)